doc/widgets/networktwitter.rst

```diff
@@ -0,0 +1,30 @@
+================
+Twitter User Graph
+================
+
+.. figure:: icons/network-twitter.png
```

Comment:
I volunteer to fix the documentation in one shot when doing the rest for Network.